Library/Client.ts

```diff
@@ -44,8 +44,11 @@ class Client {
      * @param   name    A string to identify this event in the portal.
      * @param   properties  map[string, string] - additional data used to filter events and metrics in the portal. Defaults to empty.
      * @param   measurements    map[string, number] - metrics associated with this event, displayed in Metrics Explorer on the portal. Defaults to empty.
+     * @param   tagOverrides the context tags to use for this telemetry which overwrite default context values
```

Comment:
Perhaps something like "An event-specific context that will be passed to telemetry processors handling this event before it is sent. For a context spanning your entire operation, consider appInsights.getCorrelationContext"